docs/getting-started/elastic-endpoint-reqs.asciidoc

```diff
@@ -9,4 +9,15 @@
 To properly deploy {elastic-endpoint} without a Mobile Device Management (MDM) profile, you must manually enable additional permissions on the endpoint before {elastic-endpoint} can be fully functional. For more information, refer to the instructions for your macOS version:
 
 * <<deploy-elastic-endpoint>>
-* <<deploy-elastic-endpoint-ven>>
\ No newline at end of file
+* <<deploy-elastic-endpoint-ven>>
+
+[discrete]
+== Minimum system requirements
+
+[width="100%",options="header"]
+|===
+|CPU |Disk space |RSS memory size
```

Comment:
Spell out "RSS"? Or explain it elsewhere, like an intro to the table, since a table heading has limited space.